ArticleTemplates/assets/js/modules/comments.js

```diff
@@ -266,6 +270,15 @@ define([
         }
     }
 
+    function showStaffBadges() {
```

Comment:
So `showStaffBadges` sets `display: none` on all `.comment__role` elems where `getAttribute('data-role')` is falsy? Could you just do this with css instead by using a data-attribute selector? eg.
css
```
.comment__role[data-role=''] {
  display: none;
}
 ```

Then you could remove this JS.